tests/libcxx/expected_results.txt

```diff
@@ -654,6 +654,10 @@ std/language.support/support.limits/support.limits.general/chrono.version.pass.c
 # "should not be defined when TEST_HAS_BUILTIN(__builtin_addressof) || TEST_GCC_VER >= 700 is not defined!"
 std/language.support/support.limits/support.limits.general/memory.version.pass.cpp FAIL
 
+# libc++ doesn't implement P2186R2 Removing Garbage Collection Support

```

Comment:
should add a TODO to update these expected fails once the llvm test suite has figure out we don't need pointer safety in c++23

Reply:
This should be a transition comment with the LLVM bugnumber of their task to implement this paper.